### DISCUSSION TITLE
Log Out, Reboot and Shutdown not working

### DISCUSSION THREAD

**Discussion by Cardellino** [linuxdeepin/developer-center#10768] [Troubleshooting | 故障排查]
Since Kernel 6.11.3-arch1-1 update (Archlinux + Deepin v23) the following no longer works: Log Out, Reboot and Shutdown not working.
I can only restart or shut down the computer via the console.
According to the link https://github.com/linuxdeepin/developer-center/issues/1667 the noacpi entry in grub.cfg should work again.
But it doesn't work for me.

Thank you for your support.

**Comment by BLumia** [comment]
What will happen after you click these entries? Are there any weird logs in journal?

**Reply by Cardellino** [reply]
the screen freezes.
After that I can only control the computer via console

![screen](https://github.com/user-attachments/assets/fd04dc53-e158-4f74-b03c-2e1bb23b55c8)

**Reply by BLumia** [reply]
> After that I can only control the computer via console

Yeah, at that moment, is there any suspicious error log in your systemd journal log?

**Reply by Cardellino** [reply]
Hello Blumia, the image freezes for 20 seconds and then the computer shuts down.
I will reinstall Archlinux.
You can close the ticket.
Thanks

**Reply by BLumia** [reply]
Thanks for the update. I also tested it under a clean Arch Linux installation with testing repo enabled, seems working fine on my end. If it still happens under a clean installation, feel free to let us know.
